counter/src/Main.gren

```diff
@@ -31,14 +35,19 @@ update msg model =
 
 
 view : Model -> Html Msg
-view model = 
-    Html.div []
-        [ Html.span 
-            [ Attribute.id "count" ]
-            [ Html.text <| String.fromInt model ]
+view model =
+    Html.div
+        []
+        [ Html.span
+            [ Attribute.id "count"
+            ]
+            [ Html.text
+                <| String.fromInt model
+            ]
```

Comment:
Here, I think it makes sense to format as `Html.text <| String.fromInt model`.

Reply:
:+1: this is part of retaining linebreak information, noted in https://github.com/gren-lang/example-projects/pull/11#discussion_r1037668980 re "(1)"